### PR TITLE
[PLAT-1541] Orthographic transform widget support

### PR DESCRIPTION
**Pull request by jdm717**
## Summary

Adds support for the transform widget in orthographic mode.

## Test Plan

- Test the widget in orthographic mode

## Release Notes

N/A

## Possible Regressions

Orthographic ray generation

## Dependencies

N/A

### CODE REVIEW

**Review by mikemccarty-vertex — APPROVED** [review]

File: packages/viewer/src/lib/transforms/hits.ts [hunk 1]
```diff
@@ -49,5 +53,7 @@ export function testTriangleMesh(
 
   const r = Vector3.dot(edge2, q) / det;
 
-  return !isNaN(r) && r > epsilon;
+  // Ignore the case where the computed hit position is negative
+  // if in orthographic, as the near plane can be a negative value.
+  return !isNaN(r) && (r > 0 || frame.scene.camera.isOrthographic());
```
Comment (mikemccarty-vertex) [inline packages/viewer/src/lib/transforms/hits.ts:58]: A better solution would be if `transformPointToOrthographicRay()` constructed a ray where the origin was at the near plane.

**Review by mikemccarty-vertex — APPROVED** [review]
I have some minor reservations about the logic commented (I have a feeling it could cause subtle problems later) but not enough to hold up progress.

File: packages/viewer/src/lib/transforms/hits.ts (hunk 1 above)
Comment (jdm717) [reply packages/viewer/src/lib/transforms/hits.ts:58]: I actually do think that should be the case, so thinking about this again, I'm not sure my comment here is accurate. I'll update my comment to reflect that it's a bit of a hack to correct the orthographic case where it's moving toward the camera, and create a ticket to revisit this logic

Comment (jdm717) [reply packages/viewer/src/lib/transforms/hits.ts:60]: https://vertexvis.atlassian.net/browse/PLAT-1549

**Review by mikemccarty-vertex — APPROVED** [review]

**Pull request merged**